Source/DataModels/Windows/EditorWindows/WindowHierarchy.cpp

```diff
@@ -21,264 +21,259 @@ WindowHierarchy::~WindowHierarchy()
-                }
-            }
-
-            if (ImGui::MenuItem("Move Down"))
-            {
-                if (!children.empty()
-                    && children[children.size() - 1] != gameObject)
-                {
-                    gameObject->GetParent()->MoveDownChild(gameObject);
-                }
-            }
-        }
-
-        if (gameObject != App->scene->GetLoadedScene()->GetRoot() &&
-            gameObject != App->scene->GetLoadedScene()->GetAmbientLight() &&
-            gameObject != App->scene->GetLoadedScene()->GetDirectionalLight())
-        {
-            if (ImGui::MenuItem("Delete"))
-            {
-                if (gameObject == App->scene->GetSelectedGameObject())
-                {
-                    App->scene->SetSelectedGameObject(gameObject->GetParent()); // If a GameObject is destroyed, 
-                                                                                // change the focus to its parent
-                    App->scene->GetLoadedScene()->GetRootQuadtree()->
-                        RemoveGameObjectAndChildren(gameObject->GetParent());
-                }
-                App->scene->GetLoadedScene()->GetRootQuadtree()->RemoveGameObjectAndChildren(gameObject);
-                App->scene->GetLoadedScene()->DestroyGameObject(gameObject);
-
-                ImGui::EndPopup();
-                ImGui::PopID();
-                if (nodeDrawn) // If the parent node is correctly drawn, draw its children
-                {
-                    ImGui::TreePop();
-                }
-                return;
-            }
-        }
-
-        ImGui::EndPopup();
-    }
-    ImGui::PopID();
-
-    if (gameObject != App->scene->GetLoadedScene()->GetRoot()) // The root cannot be moved around
-    {
-        if (ImGui::BeginDragDropSource())
-        {
-            UID thisID = gameObject->GetUID();
-            ImGui::SetDragDropPayload("HIERARCHY", &thisID, sizeof(UID));
-
-            ImGui::EndDragDropSource();
-        }
-    }
-
-    if (ImGui::BeginDragDropTarget())
-    {
-        if (const ImGuiPayload* payload = ImGui::AcceptDragDropPayload("HIERARCHY"))
-        {
-            UID draggedGameObjectID = *(UID*)payload->Data; // Double pointer to keep track correctly
-                                                            // of the UID of the dragged GameObject
-            GameObject* draggedGameObject =
-                App->scene->GetLoadedScene()->SearchGameObjectByID(draggedGameObjectID);
-            if (draggedGameObject)
-            {
-                draggedGameObject->MoveParent(gameObject);
-            }
-        }
-
-        ImGui::EndDragDropTarget();
-    }
-
-    if (nodeDrawn) // If the parent node is correctly drawn, draw its children
-    {
-        for (GameObject* child : children)
-        {
-            DrawRecursiveHierarchy(child);
-        }
-        ImGui::TreePop();
-    }
+	assert(gameObject);
+
+	// Delete a GameObject with the SUPR key
```

Comment:
```suggestion
	// Delete a GameObject with the `Delete / Supr` key
```

Supr key is for Spanish keyboard, other keyboard have `Delete`